browser/resources/settings/brave_leo_assistant_page/model_config_ui.ts

```diff
@@ -166,37 +165,37 @@ export class ModelConfigUI extends ModelConfigUIBase {
   }
 
   onModelLabelChange_(e: any) {
-    this.label = e.target.value
+    this.label = e.value
   }
 
   onModelRequestNameChange_(e: any) {
-    this.modelRequestName = e.target.value
+    this.modelRequestName = e.value
   }
 
   onModelSystemPromptChange_(e: any) {
-    this.modelSystemPrompt = e.target.value
+    this.modelSystemPrompt = e.value;
```

Comment:
mega nit: for consistency lets remove this
```suggestion
    this.modelSystemPrompt = e.value
```

Reply:
Nice catch. I found a few more that also needed to be removed. I personally have never settled in either camp when it comes to semicolons--sometimes I love 'em, sometimes I love that you can largely forget 'em 😀

Reply:
Personally I prefer having them :D The Brave style is to omit them though :cry:

Reply:
More often than not, I am in favor of explicitly including them. Expect more commits from me where they are sparse and inconsistently applied 😉